app/components/hook-switch.js

```diff
@@ -1,25 +1,24 @@
 import Ember from 'ember';
 
-const { alias } = Ember.computed;
-
 export default Ember.Component.extend({
   tagName: 'a',
   classNames: ['switch--icon'],
-  classNameBindings: ['active'],
-  active: alias('hook.active'),
+  classNameBindings: ['hook.active:active', 'disabled:disabled', 'disabled:inline-block'],
```

Comment:
I think we can shorten `'disabled:disabled'` to just `'disabled'`. If the value is a boolean (which we're implying when using the `:` syntax, then it will simply take the name of the property, so the `disabled` class would be applied.